lib/datadog/core/configuration/options.rb

```diff
@@ -84,6 +84,12 @@ def option_defined?(name)
             self.class.options.key?(name)
           end
 
+          def default_option?(option)
```

Comment:
Minor: Should this be `using_default?` or `default_value?`? `default_option?` sounds a bit weird to me because it seems to be asserting on the option, rather than what it is set to.